tests/layer_tests/tensorflow_tests/test_tf_Cumsum.py

```diff
@@ -54,3 +54,66 @@ def test_cumsum_basic(self, params, exclusive, reverse, ie_device, precision, ir
         self._test(*self.create_cumsum_net(**params, exclusive=exclusive, reverse=reverse),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+
+class TestComplexCumsum(CommonTFLayerTest):
+    # input_shape - should be an array
+    # axis - array which points on axis for the operation
+    # exclusive - enables exclusive Cumsum
+    # reverse - enables reverse order of Cumsum
+    def _prepare_input(self, inputs_info):
+        rng = np.random.default_rng()
+        assert 'x_real:0' in inputs_info
+        assert 'x_imag:0' in inputs_info
+        x_shape = inputs_info['x_real:0']
+        inputs_data = {}
+
+        inputs_data['x_real:0'] = 4 * rng.random(x_shape).astype(np.float64) - 2
+        inputs_data['x_imag:0'] = 4 * rng.random(x_shape).astype(np.float64) - 2
+
+        return inputs_data
+
+    def create_cumsum_net(self, input_shape, axis, exclusive, reverse):
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            x_real = tf.compat.v1.placeholder(tf.float32, input_shape, 'x_real')
+            x_imag = tf.compat.v1.placeholder(tf.float32, input_shape, 'x_imag')
+
+            complex_input = tf.complex(x_real, x_imag)
+
+            tf_axis = tf.constant(axis, dtype=tf.int32)
+            result = tf.raw_ops.Cumsum(x=complex_input, axis=tf_axis, exclusive=exclusive, reverse=reverse)
+
+            tf.compat.v1.global_variables_initializer()
+            real = tf.raw_ops.Real(input=result)
+            img = tf.raw_ops.Imag(input=result)
+
+            tf_net = sess.graph_def
```

Comment:
```suggestion
            x_real = tf.compat.v1.placeholder(tf.float32, input_shape, 'x_real')
            x_imag = tf.compat.v1.placeholder(tf.float32, input_shape, 'x_imag')
            complex_input = tf.complex(x_real, x_imag)
            tf_axis = tf.constant(axis, dtype=tf.int32)
            result = tf.raw_ops.Cumsum(x=complex_input, axis=tf_axis, exclusive=exclusive, reverse=reverse)
            real = tf.raw_ops.Real(input=result)
            img = tf.raw_ops.Imag(input=result)

            tf.compat.v1.global_variables_initializer()
            tf_net = sess.graph_def
```
            tf_net = sess.graph_def
```